FixCode/FixCode.m

```diff
@@ -11,6 +11,18 @@
 #import "Aspects.h"
 #import "FixCode.h"
 
+@interface IDEEnhancedProvisioningSigningIdentity : NSObject
```

Comment:
I couldn't find runtime headers for the views here. Should I move these somewhere else or are they fine here?